src/App.js

```diff
@@ -1,10 +1,15 @@
 import React, { Component } from "react";
-import Header from './Components/Header';
+// Import routing components
```

Comment:
we don't need any comments as a general rule for egghead code

Reply:
Okay. I'll Strip them away in the next PR.